tests/contrib/test_search.py

```diff
@@ -0,0 +1,36 @@
+from pytube import Search
+
+
+def test_for_completion_suggestions():
+
+    temp_0 = "#crapsnation"  # check missing hashtagTileRenderer object
+    try:
+        s = Search(temp_0)
+        if 'Unexpected renderer encountered.' not in s.completion_suggestions:
+            assert True
+    except KeyError:
+        assert False
+
+    temp = "crapsnation"  # check for words without suggestions
+    try:
+        s = Search(temp)
+        if 'Unexpected renderer encountered.' not in s.completion_suggestions:
+            assert True
+    except KeyError:
+        assert False
+
+    temp_1 = "gone with the wind"  # check missing movieRenderer object
+    try:
+        s = Search(temp_1)
+        if 'Unexpected renderer encountered.' not in s.completion_suggestions:
+            assert True
+    except KeyError:
+        assert False
+
+    temp_2 = "rangmnam style"  # check missing showingResultsForRenderer object
+    try:
+        s = Search(temp_2)
+        if 'Unexpected renderer encountered.' not in s.completion_suggestions:
+            assert True
+    except KeyError:
+        assert False
```

Comment:
Maybe it's just me, but I would suggest rewriting these tests. 
They are non-deterministic since they fetch data over the wire, that's why using mocks (and fixtures) is important here.

Reply:
Hey, @glubsy thanks for your input. Alright, I'll try spending some time over the weekend to get this done. I'm having a hard time trying to create mocks for this use case, but I will try again. Thanks!